pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -20,8 +27,9 @@ stdenvNoCC.mkDerivation {
     # the installation order of font files with the same name is not
     # fixed.
     rm -rv ofl/cabincondensed \
-           ofl/signikanegative \
-           ofl/signikanegativesc
+      ofl/signikanegative \
+      ofl/signikanegativesc \
+      axisregistry/tests/data
 
     if find . -name "*.ttf" | sed 's|.*/||' | sort | uniq -c | sort -n | grep -v '^.*1 '; then
```

Comment:
uniq can also sort

Reply:
How? I can't see it in the manpage.

Reply:
I think you meant sort can also uniqui-fy, but it can't do the count thing iirc

Reply:
Oh, yeah